app/views/forever_style_guide/sections/components/_hero_simple.erb

```diff
@@ -0,0 +1,20 @@
+<div class="l-section">
+  <p><code>.hero_simple</code> aims to simplify our hero treatments by providing a simple, high-contrast block for text, and the ability to easily include a colorful header without requiring an image.</p>
+  <p><strong>Keep your hero text short and clear.</strong> There will be plenty of room below the hero for details, where they're more likely to be effective.</p>
+  <p>Check out the <%= link_to "Hero Simple Full Screen Demos", demo_link('hero_simple') %> for visual reference.</p>
+</div>
+
+<hr>
+
+<div class="hero_simple hero_simple-sample">
```

Comment:
I was under the impression .hero_simple should be entirely separate, and that once we reworked the existing pages .hero wouldn't be used (unless we wanted to rename hero_simple to simplify class names).